src/App.jsx

```diff
@@ -9,7 +10,10 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <Article title={article.title} date={article.date} text={article.text} />
```

Comment:
Remember what we do when we have more than 2 attributes in a row
```suggestion
    <Article
      title={article.title}
      date={article.date}
      text={article.text} 
    />
```